test/unit/clickable-component.test.js

```diff
@@ -1,39 +1,44 @@
+/* eslint-env qunit */
 import ClickableComponent from '../../src/js/clickable-component.js';
 import TestHelpers from './test-helpers.js';
 
-q.module('ClickableComponent');
+QUnit.module('ClickableComponent');
 
-q.test('should create a div with role="button"', function(){
-  expect(2);
+QUnit.test('should create a div with role="button"', function() {
+  QUnit.expect(2);
 
-  let player = TestHelpers.makePlayer({});
+  const player = TestHelpers.makePlayer({});
 
-  let testClickableComponent = new ClickableComponent(player);
-  let el = testClickableComponent.createEl();
+  const testClickableComponent = new ClickableComponent(player);
+  const el = testClickableComponent.createEl();
 
-  equal(el.nodeName.toLowerCase(), 'div', 'the name of the element is "div"');
-  equal(el.getAttribute('role').toLowerCase(), 'button', 'the role of the element is "button"');
+  QUnit.equal(el.nodeName.toLowerCase(), 'div', 'the name of the element is "div"');
+  QUnit.equal(el.getAttribute('role').toLowerCase(), 'button',
+    'the role of the element is "button"');
 
   testClickableComponent.dispose();
   player.dispose();
 });
 
-q.test('should be enabled/disabled', function(){
-  expect(3);
+QUnit.test('should be enabled/disabled', function() {
+  QUnit.expect(3);
 
-  let player = TestHelpers.makePlayer({});
+  const player = TestHelpers.makePlayer({});
 
-  let testClickableComponent = new ClickableComponent(player);
+  const testClickableComponent = new ClickableComponent(player);
 
-  equal(testClickableComponent.hasClass('vjs-disabled'), false, 'ClickableComponent defaults to enabled');
+  QUnit.equal(testClickableComponent.hasClass('vjs-disabled'), false,
```

Comment:
On second thought, all those comments seem like a bit of a waste. I'll just call out a few places.